src/main/java/nl/tudelft/dnainator/ui/DNAinator.java

```diff
@@ -24,20 +26,31 @@
 	private static final int MIN_WIDTH = 300;
 	private static final int MIN_HEIGHT = 150;
 
+	private Stage stage;
```

Comment:
You need a reference _because_ you set the dimensions in a separate method.
There is however no need for the separate method in the first place.